tests/e2e/env/utils/test-config.js

```diff
@@ -17,9 +17,14 @@ const getTestConfig = () => {
 	const rawTestConfig = fs.readFileSync( testConfigFile );
 
 	let testConfig = JSON.parse(rawTestConfig);
-	testConfig.baseUrl = testConfig.url.substr(0, testConfig.url.length - 1);
 	let testPort = testConfig.url.match(/[0-9]+/);
-	testConfig.port = testPort[0] ? testPort[0] : '8084';
+	if ( Array.isArray( testPort ) ) {
+		testConfig.baseUrl = testConfig.url.substr(0, testConfig.url.length - 1);
+		testConfig.port = testPort[0] ? testPort[0] : '8084';
+	} else {
+		testConfig.baseUrl = testConfig.url;
```

Comment:
We need to remove the trailing slash in this case as well, otherwise configurations where it matters will fall over. My local environment for instance fails because when checking for Docker status it queries `http://woocommerce.test//?pagename=ready` and doesn't like the double slash.

```suggestion
		testConfig.baseUrl = testConfig.url.substr(0, testConfig.url.length - 1);
```

Reply:
I tested this with multiple hosts without a port and it worked at time of testing. I'll re-check this.